app/server/utils/executor.py

```diff
@@ -0,0 +1,24 @@
+from typing import Callable, List, Dict, Optional
+
+from flask import g
+
+
+def add_after_request_executor_job(
+        fn: Callable,
+        args: Optional[List] = None,
+        kwargs: Optional[Dict] = None
+):
+    """
+    Utility function for adding executor jobs that need to be submitted AFTER a request has completed
+    (generally speaking, to avoid potential race conditions).
+
+    :param fn: the executor decorated function that will be submitted, eg do_foo NOT do_foo.submit
+    :param args: function arguments
+    :param kwargs: function keyword arguments
+    :return:
+    """
+
+    args = args or []
```

Comment:
Could you set these default values on lines 8/9 (in place of `None`)?

Reply:
its_a_trap.jpg
https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Huh!